bubble-wrap-style.yaml

```diff
@@ -5120,6 +5036,19 @@ layers:
                     color: [0.765,0.765,0.765]
                     visible: true
 
+#        # (20190729: nvk) TODO: This should only show when no other kind condition
```

Comment:
I believe this kind of "else" condition can be done with the new `exclusive` layer property added in JS -- make all the other layers `exclusive`, and if none matches, the non-exclusive "else" one does -- but it's not in ES yet.

Reply:
YES! `exclusive` would be great here :) So in Tangram ES the layer won't match and won't display? Seems backwards compatible and okay to implement?

Reply:
Reading the docs in https://github.com/tangrams/tangram/pull/705 I'm confused how I'd implement this. Is there a `!exclusive` or `priority: -1` to mean evaluate this one layer last? Or do I have to add priority to every other sibling layer here, too?